app/components/AddNewParticipant/ParticipantForm.js

```diff
@@ -46,6 +46,15 @@ class ParticipantForm extends Component {
     return !this.state.isValidAge ? 'red' : '';
   }
 
+  closeSelectBox = (index) => {
```

Comment:
@limkimsan params should be except_index -> closeSelectBox(except_index)

Reply:
Bong, I have changed the variable name.